projects/packages/post-list/src/class-admin.php

```diff
@@ -0,0 +1,228 @@
+<?php
+/**
+ * The PostList Admin Area.
+ *
+ * @package automattic/jetpack-post-list
+ */
+
+namespace Automattic\Jetpack\PostList;
+
+use Automattic\Jetpack\Assets;
+
+/**
+ * The PostList Admin Area
+ */
+class Admin {
+
+	/**
+	 * Construction.
+	 */
+	public function __construct() {
+		if ( ! did_action( 'jetpack_on_posts_list_init' ) ) {
+			add_filter( 'query_vars', array( $this, 'add_query_vars' ) );
+			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+
+			/**
+			 * Action called after initializing PostList Admin resources.
+			 *
+			 * @since 9.8.0
```

Comment:
Let's use `$$next-version$$` here so the version can be generated automatically:

```suggestion
			 * @since $$next-version$$
```

--
https://github.com/Automattic/jetpack/blob/bce4852759f6d2c3f9b1e84b792be9988a031fb1/projects/packages/README.md#package-version-annotations